docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -436,7 +436,23 @@ The connector supports pushdown for a number of operations:
 
 .. include:: join-pushdown-enabled-true.fragment
 
-.. include:: no-pushdown-text-type.fragment
+Predicate pushdown support
+""""""""""""""""""""""""""
+
+The connector supports pushdown of any predicates on ``VARCHAR`` and
+``NVARCHAR`` columns. The following operators are supported:
+
+- EQUAL
+- NOT EQUAL
+- IN
+- NOT IN
+
+.. code-block:: sql
+
+.. example
+
+To enable pushdown for case-sensitive columns, change the collation setting to a
```

Comment:
I have no idea what that means. What setting do you change from what to what?

Reply:
In SqlServer every textual column have a Collation.
Collation it's kind of set of rules how to treat text symbols in this column during different operations.
For example collation Latin1_General_CI_AS - means for this column we translate bytes to symbols using Latin byte to symbol translation and CI -  mean Case Insensitivity and other rules.
So our code is working only if collation is Case Sensitive for column.

Reply:
Case sensitive collation means that in Collation name there is part like _CS_ or _BIN_ or _BIN2_